tests/data_context/test_base_data_context.py

```diff
@@ -0,0 +1,160 @@
+import os
+from typing import List
+
+import pytest
+
+from great_expectations.core.yaml_handler import YAMLHandler
+from great_expectations.data_context import BaseDataContext
+from great_expectations.data_context.types.base import DataContextConfig
+
+from great_expectations.data_context.store import (  # isort:skip
+    ExpectationsStore,
+    ValidationsStore,
+    EvaluationParameterStore,
+)
+
+yaml: YAMLHandler = YAMLHandler()
+
+
+@pytest.fixture()
+def basic_in_memory_data_context_config_just_stores():
+    return DataContextConfig(
+        config_version=2.0,
```

Comment:
Should this be tested on v3.0?
```suggestion
        config_version=3.0,
```

Reply:
you're absolutely right. Thanks for catching this